addons/web_editor/static/src/js/backend.js

```diff
@@ -187,6 +187,18 @@ var FieldTextHtmlSimple = basic_fields.DebouncedField.extend({
         }
         return value;
     },
+
+    _renderTranslateButton: function () {
+        if (_t.database.multi_lang && this.field.translate && this.res_id) {
+            $(QWeb.render('web_editor.FieldTextHtml.button.translate', {'widget': this}))
+                .appendTo(this.$('.note-toolbar'))
+                .on('click', this._onTranslate.bind(this));
+        }
+    },
+
+    _onTranslate: function () {
```

Comment:
This should be in 'Handlers' section (see https://github.com/odoo/odoo/blob/master/addons/web/static/src/js/fields/abstract_field.js#L395 for example). New coding guidelines: split functions as follows (in that order): first unnamed section containing lifecyle widget functions (init, willStart, start, destroy), then 'Public' section containing all public functions (called by the outside of widget, not prefixed by _), then 'Private' section containing all non event handlers private functions (only called by the widget itself, prefixed by _), and finally 'Handlers' section containing all event handlers (prefixed by _on, as you did). Also document all your functions. Again, you can see AbstractField for example. Tag your private functions with `@private`. Explanations of what the function does is not mandatory if it is straightforward (e.g. for your event handler, it's not necessary, so basically the tag `@private` is enough.